### PR TITLE
Various minor cleanups

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by ids1024** [comment]
This appears to revert the `gir` submodule to an earlier commit. I assume that was committed unintentionally?

**Comment by sdroege** [comment]
> This appears to revert the `gir` submodule to an earlier commit. I assume that was committed unintentionally?

Yes, I hate git submodules :) Thanks for noticing!

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by sdroege** [comment]
Once https://github.com/gtk-rs/gir/pull/985 is in and we regenerate with that, we won't have any compiler warnings left with latest stable and nightly. How it should've always been.

Can we have a github action that complains about warnings (makes them errors) and annotates PRs with them somehow? @bilelmoussaoui ?

**Comment by bilelmoussaoui** [comment]
Definitely, I will handle that once my other CI additions are in

File: atk/src/lib.rs [hunk 1]
```diff
@@ -32,6 +32,7 @@ mod rt;
 #[cfg_attr(feature = "cargo-clippy", allow(let_and_return))]
 #[cfg_attr(feature = "cargo-clippy", allow(many_single_char_names))]
 #[cfg_attr(feature = "cargo-clippy", allow(wrong_self_convention))]
+#[allow(unused_imports)]
```
Comment (GuillaumeGomez) [inline atk/src/lib.rs:35]: At some point it'd be nice if `gir` imports were fixed. :)

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

File: atk/src/lib.rs (hunk 1 above)
Comment (sdroege) [reply atk/src/lib.rs:35]: Indeed, but that's quite non-trivial to get right :)

**Pull request merged**